java/app/grapheneos/setupwizard/action/SetupWizard.kt

```diff
@@ -95,7 +95,11 @@ object SetupWizard {
         return PartnerConfigHelper.getSuwDefaultThemeString(appContext)
     }
 
+    // primary user here means the first user of the device, the user that is implicitly created
+    // on first boot.
+    // any other subsequent users are secondary users
     val isPrimaryUser: Boolean by lazy {
         appContext.getSystemService(UserManager::class.java).isSystemUser
     }
+    val isSecondaryUser = !isPrimaryUser
```

Comment:
This declaration makes `lazy` above useless, since `isPrimaryUser` is immediately evaluated.

`UserManager.isSystemUser()` call is cheap anyway, there's no need to load it lazily.

Reply:
made `isSecondayUser` lazy as well, thanks for pointing out!
yes the call is cheap, and another reason for declaring lazy is to prevent mishappenings due to an out-of-order class loading, although that's very rare, but just in case.